pyproject.toml

```diff
@@ -23,7 +23,7 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "^3.9"
-grpcio = "^1.48.0"
+grpcio = "^1.47.0"
```

Comment:
The grpcio dependencies are mostly for proto-gen. To see if this is possible, we'd have to change the dependency in [sdk-proto-gen](https://github.com/NibiruChain/sdk-proto-gen) and see if the code generation is unaffected.

Reply:
I'll try it out now